plugins/destination/postgresql/README.md

```diff
@@ -1,6 +1,6 @@
 # CloudQuery PostgreSQL Destination Plugin
 
-This destination plugin lets you sync data from a CloudQuery source to a PostgreSQL compatible database.
+This destination plugin lets you sync data from a CloudQuery source to a PostgreSQL compatible database (CockroachDB).
```

Comment:
```suggestion
This destination plugin lets you sync data from a CloudQuery source to a PostgreSQL or CockroachDB database.
```

We haven't tested with others. Even CockroachDB differs a bit from PostgreSQL I think

Reply:
We have tested on RDS for example and on Google Cloud SQL, so it is PostgreSQL compatible (maybe we need to adds RDS as well to the parentheses )

Reply:
Nice, so maybe do `e.g. CockroachDB, RDS, Google Cloud SQL` like you said